packages/docs/.vuepress/plugins/plugin-micromodal/clientRootMixin.js

```diff
@@ -2,11 +2,13 @@ module.exports = {
   updated() {
     import('micromodal').then((module) => {
       const MicroModal = module.default;
-      MicroModal.init({
-        awaitCloseAnimation: true,
-        disableScroll: true,
-        disableFocus: true
-      });
+      setTimeout(()=> {
```

Comment:
Why?

Reply:
cc @arnoldsandoval-okta

Reply:
It's hacky for sure @santhoshbalakrishnan and on my list to fix. Due to the way Vue rendered the page, it wasn't instantiating the external dependency on `mount,` nor did it happen on `update`. The set timeout merely ensures that the dom nodes required by the dependencies present at the time of instantiation.